shared/css/teacher-dashboard.scss

```diff
@@ -310,23 +310,57 @@ td.studentname {
 .signin_card dl dd {
 }
 
-.teacher-announcements {
-  background-color: #e7e8ea; // lightest gray
```

Comment:
color.scss is included, so you can replace these colors with their variable names.